tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -86,19 +85,19 @@ def test__create__ok(self):
                 )
 
                 dataset_id = dataset.id
-                artifact_ids = get_ids(dataset.artifacts)
-                deployment_directory_ids = get_ids(dataset.deployment_directories)
-                contributor_ids = get_ids(dataset.contributors)
+                artifact_ids = [i.id for i in dataset.artifacts]
+                deployment_directory_ids = [i.id for i in dataset.deployment_directories]
+                contributor_ids = [i.id for i in dataset.contributors]
 
                 # Expire all local objects and retrieve them from the DB to make sure the transactions went through.
                 Dataset.db.session.expire_all()
 
                 dataset = Dataset.get(dataset_id)
```

Comment:
sugg: use clearer variable name to differentiate from created Dataset. e.g. `dataset` -> `dataset_from_db`

Reply:
fixed